PULL_REQUEST_TEMPLATE.md

```diff
@@ -40,5 +40,5 @@ Things to consider here:
 - Backfills
 - Monitoring updates
 
-_NOTE:_ This template comes from https://github.com/zeus-health/.github/PULL_REQUEST_TEMPLATE.md.
+_NOTE:_ This template comes from [here](https://github.com/zeus-health/.github/blob/main/PULL_REQUEST_TEMPLATE.md)https://github.com/zeus-health/.github/blob/main/PULL_REQUEST_TEMPLATE.md.
```

Comment:
This is going to render like:

_NOTE:_ This template comes from [here](https://github.com/zeus-health/.github/blob/main/PULL_REQUEST_TEMPLATE.md)https://github.com/zeus-health/.github/blob/main/PULL_REQUEST_TEMPLATE.md.


I think you want to drop the second copy of the URL.

Reply:
🤦🏻‍♂️ clearly I didn't pay enough attention to the preview. thanks!